internal/runners/languages/languages.go

```diff
@@ -9,36 +9,49 @@ import (
 	"github.com/ActiveState/cli/pkg/project"
 )
 
+// Languages manages the listing execution context.
 type Languages struct {
 	out     output.Outputer
 	project *project.Project
 }
 
+// NewLanguages prepares a list execution context for use.
 func NewLanguages(prime primeable) *Languages {
 	return &Languages{
 		prime.Output(),
 		prime.Project(),
 	}
 }
 
+// Listing represents the output data of a list of languages.
 type Listing struct {
 	Languages []model.Language `json:"languages"`
 }
 
+// MarshalOutput implements the output.Marshaller interface.
 func (l Listing) MarshalOutput(f output.Format) interface{} {
 	if f == output.PlainFormatName {
 		return l.Languages
 	}
 	return l
 }
 
+// Run executes the list behavior.
 func (l *Languages) Run() error {
 	if l.project == nil {
 		return locale.NewInputError("err_no_project")
 	}
-	langs, err := model.FetchLanguagesForCommit(l.project.CommitUUID())
-	if err != nil {
-		return err
+
+	commitUUID := l.project.CommitUUID()
+	if commitUUID == "" {
+		return locale.NewError(
+			"err_languages_no_commitid", "A CommitID is required to list languages",
```

Comment:
```suggestion
			"err_languages_no_commitid", "Your activestate.yaml does not have a commit defined, you may need to run [ACTIONABLE]`state pull`[/RESET] first.",
```

Can we also add a tip here to run `state pull`?